libs/SalesforceSDK/src/com/salesforce/androidsdk/security/SalesforceKeyGenerator.java

```diff
@@ -166,15 +195,38 @@ public synchronized static void upgradeTo7Dot1() {
         }
     }
 
+    /**
+     * Upgrades the keys stored in SharedPrefs to encrypted keys. This is a one-time
+     * migration step that's run while upgrading to Mobile SDK 8.2.
+     *
+     * @deprecated Will be removed in Mobile SDK 9.0.
```

Comment:
We should keep that method longer for apps that upgrade from major to major.

Reply:
Sure - I'll bump it up to `10.0`.